src/Gui/DlgDisplayProperties.ui

```diff
@@ -531,21 +524,5 @@
     </hint>
    </hints>
   </connection>
-  <connection>
-   <sender>buttonBox</sender>
-   <signal>rejected()</signal>
```

Comment:
Why did you remove this signal?

Reply:
I removed the whole buttonBox widget. It is redundant because the same thing is already implemented in DlgDisplayPropertiesImp.cpp with:
```
QDialogButtonBox::StandardButtons TaskDisplayProperties::getStandardButtons() const
{
    return QDialogButtonBox::Close;
}
```